Add dkms support.

Joe136 opened the pull request:
Hi, I have added dkms.conf to automatically update the driver on kernel update.

In addition, this can be added to the Makefile (dkms needs to know the version).

```
DKMS = dkms

dkms_install: VERSION = $(shell grep 'PACKAGE_VERSION' dkms.conf | grep -oEe '[0-9.]+')
dkms_install:
   $(DKMS) add .
   $(DKMS) install emlog/$(VERSION)

dkms_remove: VERSION = $(shell grep 'PACKAGE_VERSION' dkms.conf | grep -oEe '[0-9.]+')
dkms_remove:
   $(DKMS) remove emlog/$(VERSION) --all
   rm -rf /usr/src/emlog-$(VERSION)
```

ahippo commented on dkms.conf:
```diff
@@ -0,0 +1,7 @@
+PACKAGE_NAME="emlog"
+PACKAGE_VERSION="0.70"
+AUTOINSTALL="yes"
+CLEAN="make clean"
```
`make clean` is assumed to be the default, right?
So, can be omitted for brevity (as `make` is), right?
Not a bit deal though.

Joe136 replied:
I copied it from nvidia driver, don't know why they wrote this line.

ahippo replied:
> I copied it from nvidia driver, don't know why they wrote this line

Could you, please, try to remove it and see if DMKS works without it?

Joe136 commented:
I added the Makefile
- in addition, I included the other targets to .PHONY
- I used MVER for the version
- **/usr/src** can be changed in **/etc/dkms/framework.conf** or `dkms --sourcetree /usr/src`
  - I'm not sure how to detect the correct path, so I commented the *rm* command

I changed dkms.conf
- MAKE_CLEAN is not needed
- I added a documentation
